instant-kraft/README.md

```diff
@@ -0,0 +1,56 @@
+# Instant KRaft Demo
+Let's make some delicious instant KRaft
+          ___
+        .'O o'-._
+       / O o_.-`|
+      /O_.-'  O |
+      |O o O .-`
+      |o O_.-'
+      '--`
+
+## Step 1
+```
+$ git clone https://github.com/confluentinc/examples.git && cd examples/instant-kraft
+
+$ docker compose up
+```
+
+That's it! One container, one process, doing the the broker and controller work. No Zookeeper.
+The key server configure to enable KRaft is the `process.roles`. Here we have the same node doing both the broker and quorum conroller work. Note that this is ok for local development, but in production you should have seperate nodes for the broker and controller quorum nodes.
+
+```
+KAFKA_PROCESS_ROLES: 'broker,controller'
+```
+
+## Step 2
+Start streaming
+
+```
+$ docker compose exec broker kafka-topics --create --topic zookeeper-vacation-ideas --bootstrap-server localhost:9092
+
+$ docker-compose exec broker kafka-producer-perf-test --topic zookeeper-vacation-ideas \
+    --num-records 200000 \
+    --record-size 50 \
+    --throughput -1 \
+    --producer-props \
+        acks=all \
+        bootstrap.servers=localhost:9092 \
+        compression.type=none \
+        batch.size=8196
+```
+
+## Step 3
+Explore the cluster using the new kafka metadata command line tool.
```

Comment:
Should we refer to it inline by name?  I don't mind, just asking: `kafka-metadata-shell`